src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,47 @@
+@use '../_variables/variables' as *;
+@use '../mixins/mixins.scss' as *;
+
+.nav {
+  &__list {
+    display: flex;
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-left: 0;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    color: inherit;
+    font-size: 12px;
+    text-transform: uppercase;
+    line-height: 60px;
+    font-weight: 500;
+
+    @include transition-options(color);
```

Comment:
The mixin 'transition-options' is used here, but it's not clear if it's defined in the 'mixins.scss' file. Ensure that this mixin is defined and correctly imported to avoid potential errors.